VERSION

```diff
@@ -1 +1 @@
-1
+2
```

Comment:
Technically, this breaks compat with any repos that depend on these workflows (including repos from sailpoint/pavilion). 

@davidtaylorhq @CvX does this make sense? This means we would need to make another mass PR to upgrade the version of the theme/plugin workflow for our own repos at some point.